code/addons/test/src/node/test-manager.ts

```diff
@@ -53,14 +53,7 @@ export class TestManager {
           coverage: this.coverage,
         });
       } catch (e) {
-        const isV8 = e.message?.includes('@vitest/coverage-v8');
-        const isIstanbul = e.message?.includes('@vitest/coverage-istanbul');
-
-        if (e.message?.includes('Error: Failed to load url') && (isIstanbul || isV8)) {
-          const coveragePackage = isIstanbul ? 'coverage-istanbul' : 'coverage-v8';
-          e.message = `Please install the @vitest/${coveragePackage} package to run with coverage`;
-        }
-        this.reportFatalError('Failed to change coverage mode', e);
+        this.reportFatalError('Failed to change coverage configuration', e);
```

Comment:
this case would already be handled _within_ the Vitest Manager, so it doesn't make sense to try and handle it here too.